### PR TITLE
add openapi

### PR DESCRIPTION
**Pull request by lvenier**
https://github.com/SaaShup/netbox-docker-agent/issues/15

### CODE REVIEW

File: openapi.yml [hunk 1]
```diff
@@ -0,0 +1,1020 @@
+info:
+  title: Untitled service
```
Comment (fanshan) [inline openapi.yml:2]:
```suggestion
  title: Netbox Docker Agent
```

**Review by fanshan — CHANGES_REQUESTED** [review]
Since there is a lot of endmpoints, can you split its by tag?

File: openapi.yml [hunk 2]
```diff
@@ -0,0 +1,1020 @@
+info:
+  title: Netbox Docker Agent
+  version: 1.0.0
```
Comment (fanshan) [inline openapi.yml:3]:
```suggestion
  version: 0.4.3
```

**Review by fanshan — CHANGES_REQUESTED** [review]

**Comment by lvenier** [comment]
tags added

**Pull request merged**